packages/quantic/force-app/main/default/lwc/quanticCategoryFacetParent/quanticCategoryFacetParent.js

```diff
@@ -0,0 +1,14 @@
+import { LightningElement, api } from 'lwc';
+
+export default class QuanticCategoryFacetParent extends LightningElement {
```

Comment:
What exactly is the purpose of a FacetParent?

Reply:
I delete it, it s better to use `quantic-category-facet-value`